packages/extension/src/newtab/ShortcutLinks.tsx

```diff
@@ -52,84 +52,16 @@ import ShortcutOptionsMenu from './ShortcutOptionsMenu';
 const pixelRatio = globalThis?.window.devicePixelRatio ?? 1;
 const iconSize = Math.round(24 * pixelRatio);
 
-const ShortCutV1Placeholder = ({
-  initialItem = false,
-  onClick,
-}: {
-  initialItem?: boolean;
-  onClick: () => void;
-}) => {
-  return (
-    <button
-      className="group mr-4 flex flex-col items-center first:mr-2 last-of-type:mr-2 hover:cursor-pointer"
-      onClick={onClick}
-      type="button"
-    >
-      <div className="mb-2 flex size-12 items-center justify-center rounded-full bg-surface-float text-text-secondary group-first:mb-1 group-hover:text-text-primary">
-        <PlusIcon
-          className="inline group-hover:hidden"
-          size={IconSize.Size16}
-        />
-        <PlusIcon
-          className="hidden group-hover:inline"
-          secondary
-          size={IconSize.Size16}
-        />
-      </div>
-      {initialItem ? (
-        <span className="text-text-tertiary typo-caption2">Add shortcut</span>
-      ) : (
-        <span className="h-2 w-10 rounded-10 bg-surface-float" />
-      )}
-    </button>
-  );
-};
+interface ShortcutLinksProps {
+  shouldUseListFeedLayout: boolean;
+}
 
-const ShortcutV1Item = ({
-  url,
-  onLinkClick,
-}: {
-  url: string;
+type ActualShortcutLinksProps = {
```

Comment:
I'm a bit confused by the `Actual` vs the one above.
Why is there a difference?

Reply:
Hi @rebelchris , it is the internal type for the two versions, maybe is not the best name! `Inner` prefix should be better?

Reply:
I think maybe if you abstract into 2 files it will already solve it, right?